pallets/liquidity-rewards/src/lib.rs

```diff
@@ -62,7 +62,7 @@ use sp_runtime::{
 use sp_std::mem;
 use weights::WeightInfo;
 
-/// Type that contains the finish timestamp of an epoch
+/// Type that contains the finish timestamp of an epoch.
```

Comment:
This doc comment doesn't seem be quite what the code shows; the code tells me that `EpochTimestamp` is basically a wrapper around `BlockNumber`, not "the finish timestamp of an epoch" 🤔